.github/workflows/scripts/monitor-typescript.ts

```diff
@@ -3,14 +3,28 @@ import '@sentry/tracing';
 
 // eslint-disable-next-line import/no-nodejs-modules
 import fs from 'fs';
+// eslint-disable-next-line import/no-nodejs-modules
+import os from 'os';
 
 import * as Sentry from '@sentry/node';
+import {Event} from '@sentry/types';
+import {timestampWithMs} from '@sentry/utils';
 
 Sentry.init({
-  // jest project under Sentry organization (dev productivity team)
-  dsn: 'https://3fe1dce93e3a4267979ebad67f3de327@sentry.io/4857230',
+  // typescript-compiler project under sentry-test organization (dev productivity team)
+  dsn: 'https://ca215f623d6f456aa1d09ebc1efad79d@o19635.ingest.sentry.io/6326775',
+  // setting a tunnel implicitly sends the event as an envelope instead of json
+  tunnel: 'https://o19635.ingest.sentry.io/api/6326775/envelope/',
   tracesSampleRate: 1.0,
   environment: 'ci',
+  beforeSend: (event: any) => {
+    if (event.type === 'profile') {
+      event.profile_id = event.event_id;
```

Comment:
Why is this and the next line necessary?

Reply:
Profiles do not have first-class support in the SDK yet, so this is just an experiment to generate some data before we finalize how the SDK will handle them.